tests/integration/buildcmd/test_build_terraform_applications.py

```diff
@@ -267,22 +170,10 @@ class TestBuildTerraformApplicationsWithZipBasedLambdaFunctionAndLocalBackend(Bu
         else Path("terraform/zip_based_lambda_functions_local_backend_windows")
     )
     functions = [
-        ("module.function7.aws_lambda_function.this[0]", "hello world 7 - override version", True),
-        ("function7", "hello world 7 - override version", True),
-        ("module.function7.aws_lambda_function.this[0]", "hello world 7", False),
-        ("function7", "hello world 7", False),
-        ("module.function8.aws_lambda_function.this[0]", "hello world 8 - override version", True),
-        ("function8", "hello world 8 - override version", True),
-        ("module.function8.aws_lambda_function.this[0]", "hello world 8", False),
-        ("function8", "hello world 8", False),
```

Comment:
removed these functions from the windows integration test cases, as Serverless TF has issues while running in windows https://github.com/terraform-aws-modules/terraform-aws-lambda/issues/142

Reply:
Lets add a comment in the code as well.